providers/default/defltprov.c

```diff
@@ -144,9 +143,9 @@ static const OSSL_ALGORITHM deflt_ciphers[] = {
     { "ARIA-256-ECB", "default=yes", aria256ecb_functions },
     { "ARIA-192-ECB", "default=yes", aria192ecb_functions },
     { "ARIA-128-ECB", "default=yes", aria128ecb_functions },
-    { "ARIA-256-CBC", "default=yes", aria256cbc_functions },
-    { "ARIA-192-CBC", "default=yes", aria192cbc_functions },
-    { "ARIA-128-CBC", "default=yes", aria128cbc_functions },
+    { "ARIA-256-CBC:ARIA256", "default=yes", aria256cbc_functions },
```

Comment:
Why this alias?

Reply:
Because it's already there:

```
: ; openssl list -cipher-algorithms|grep -i aria
...
aria128 => ARIA-128-CBC
aria192 => ARIA-192-CBC
aria256 => ARIA-256-CBC
```

Reply:
Ok. Weird - but ok.

Reply:
There's a *lot* of weirdness in the naming scheme.  Have you read through `crypto/objects/objects.txt`?  It's an interesting read, and even more so when you realise that sometime through history, ASN.1 modules started using that `id-` prefix we all love so much...  this has also affected our naming...  and then we have TLS cipher suite naming vs naming in ASN.1 modules, where one has things like "AES256" while the other has things like "AES-256".

Let's just say that it's a tad messy, and we have adapted to all, or partly so, all depending who implemented the affected algorithm.

Reply:
This one would be my fault and due to my ignorance at the time.